### DISCUSSION TITLE
Is it possible to configure fantomas' path?

### DISCUSSION THREAD

**Discussion by ratsclub** [ionide/ionide-vscode-fsharp#2025] [Q&A]
I would like to set a specific path for fantomas but I coulnd't find an option for this on the extension settings. Is it possible?

**Comment by baronfel** [comment]
It's not possible today to customize the paths - FSAC uses Fantomas from either a local .NET Tool install (one you have in the .config/dotnet-tools.json file) or a global .NET Tool install. That's all controlled by the Fantomas.Client library from the https://github.com/fsprojects/fantomas repository - if more configuration was added to the Fantomas.Client library then we could surface some of that config in the FSAC config though.

**Reply by ratsclub** [reply]
Oh, that's sad. I filled a feature request there too https://github.com/fsprojects/fantomas/issues/3099. I might be able to implement it in both projects if I get the green light!

The main reason for this request is to package ionide with fantomas bundled for Nix users.
